IntelliTect.TestTools.TestFramework/ExampleTests/ExampleTests/ExampleTests.csproj

```diff
@@ -2,16 +2,29 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
-
+    <LangVersion>9.0</LangVersion>
```

Comment:
As per discussion would be nice to pub this up if it is a simple change. I would also bump the LangVersion to 11.